moveit_setup_assistant/src/tools/collision_linear_model.cpp

```diff
@@ -239,6 +246,9 @@ void SortFilterProxyModel::setShowAll(bool show_all)
 bool SortFilterProxyModel::filterAcceptsRow(int source_row, const QModelIndex& source_parent) const
 {
   CollisionLinearModel* m = qobject_cast<CollisionLinearModel*>(sourceModel());
+  if (!m)
+    return false;
```

Comment:
Same here: You catch the invalid pointer, but you return an arbitrary value that changes the behavior of the program without indicating the error (by a segfault).